### PR TITLE
Merge backend into repo

### PR DESCRIPTION
**Pull request by YouriRombouts**

### CODE REVIEW

**Review by BrickPi — APPROVED** [review]
I'm not seeing anything glaring, so I'm going to approve it but wait until Wednesday to merge it just in case we get another person familiar with ASP to check it.

**Review by ClarkeCode — APPROVED** [review]

**Review by ClarkeCode — APPROVED** [review]

**Comment by Spartan322** [comment]
Not sure why I was requested to review, I don't really know much about ASP, I ain't much of a web development guy.

**Pull request merged**
